### DISCUSSION TITLE
Cast birdseye camera to Chromecast

### DISCUSSION THREAD

**Discussion by norbertjoni** [blakeblackshear/frigate#5893] [Ideas]
Has anyone had success in doing it? 
I see there is a workaround with frigate card but it's not working for me. 

type: custom:frigate-card
cameras:
  - camera_name: birdseye

![image](https://user-images.githubusercontent.com/25225562/229341973-51ff8d6e-e1cd-4085-8c65-7f0c8e93de5b.jpeg)

**Comment by NickM-27** [comment]
What version of frigate are you running?

**Reply by norbertjoni** [reply]
I am on the blakeblackshear/frigate:stable image.
Probably i need to update to the beta.

**Reply by FigJam23** [reply]
Interested too. Can get birds eye  via browser addon on on Chrome cast. Or via pc in Chrome and casting screen but this is not that useful. I'm on version 13.1 not sure if there are new ways via frigate directly or autimation in hassio could maybe work 🤔
